### PR TITLE
Sync Serverless CI config/eventing-hyperfoil-benchmark.yaml

### PR DESCRIPTION
**Pull request by aliok**
Based on https://github.com/openshift-knative/hack/pull/42

### CODE REVIEW

File: ci-operator/jobs/openshift-knative/eventing-hyperfoil-benchmark/openshift-knative-eventing-hyperfoil-benchmark-main-presubmits.yaml [hunk 1]
```diff
@@ -52,6 +52,71 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )412-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
```
Comment (pierDipi) [inline ci-operator/jobs/openshift-knative/eventing-hyperfoil-benchmark/openshift-knative-eventing-hyperfoil-benchmark-main-presubmits.yaml:56]: /lgtm
/approve

**Comment by aliok** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-hyperfoil-benchmark-main-412-test-kafka-8045b51-aws-ocp-412
/pj-rehearse periodic-ci-openshift-knative-eventing-hyperfoil-benchmark-main-412-test-kafka-8045b51-aws-ocp-412-continuous

**Pull request merged**